### PR TITLE
Update cve-2019-5418 w/ simple WaF secondary check

### PR DESCRIPTION
**Pull request by kingthorin**
I didn't modify changelog this time and no release has occurred.

### CODE REVIEW

**Comment by psiinon** [comment]
This pull request **introduces 2 alerts** when merging 8d5bd841193e80ca2d266488b10ed3c9e22ee513 into b485697e676819fedcb68a45ad4322e689364414 - [view on LGTM.com](https://lgtm.com/projects/g/zaproxy/community-scripts/rev/pr-85c8eab6574ac475c8041ce391328db5b5d6b9ed)

**new alerts:**

* 2 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by kingthorin** [comment]
The LGTM issues were fixed in a forced push:
image

File: active/cve-2019-5418.js [hunk 1]
```diff
@@ -26,21 +26,33 @@ data from the server file system.'
...
+    as.sendAndReceive(msg2, false, true);
 
     var re = /root:.:0:0/g
-    var body = msg.getResponseBody().toString()
+    var body = msg2.getResponseBody().toString()
 
     if (re.test(body)) {
         as.raiseAlert(alertRisk, alertConfidence, alertTitle, alertDesc, 
-          url, '', '', alertInfo, alertSolution, body.match(re), cweId, wascId, msg);
+          url, '', '', alertInfo, alertSolution, body.match(re), cweId, wascId, msg2);
```
Comment (thc202) [inline active/cve-2019-5418.js]: Should the evidence be changed to just the first match like in the added `raiseAlert`?

Comment (kingthorin) [reply active/cve-2019-5418.js]: I could change it, but I don't think there can be multiple root entries in passwd

Comment (thc202) [reply active/cve-2019-5418.js]: Right, but the array when converted to string will still have the `[]`?

Comment (kingthorin) [reply active/cve-2019-5418.js]: Hmmm it worked fine for me, but it doesn't hurt to change it. In testing I had hit a comma seperated list while working on the new addition. Tweak coming.

**Comment by kingthorin** [comment]
Tweaked

**Comment by thc202** [comment]
Thanks!

**Pull request merged**